<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryRadarWidget</class>
 <widget class="QWidget" name="HistoryRadarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>历史雷达故障信息表</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_startTime">
       <property name="text">
        <string>开始时间:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit_start">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_endTime">
       <property name="text">
        <string>结束时间:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit_end">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>所有</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_qry">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>98</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableView">
     <property name="styleSheet">
      <string notr="true">QTableView::item:selected { background-color: rgb(135,206,250) }</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
